docs/core/extensions/snippets/logging/library-authors/LibraryConfiguration.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+
+namespace Logging.LibraryAuthors;
+
+public sealed class LibraryConfiguration
+{
+    public static ILoggerFactory LoggerFactory { get; private set; } = NullLoggerFactory.Instance;
```

Comment:
If the library is going to have a public property I'm not sure why we wouldn't make it settable? My example had separate SetLoggerFactory() method because there was no public property.